### PR TITLE
feat(docs): Add working example to Connections and Requests concept page.

### PR DESCRIPTION
**Pull request by StephanieJKS**
Closes ~#ISSUE_NUMBER~

### What are the changes and their implications?
This PR aims to improve the v4 docs, by adding a working example to the Connections and Requests concept page.

## Checklist

- [x] Pull request is made to the "develop" branch
- [ ] Tests added
- [x] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/docs/concepts/connections-and-requests.yaml [hunk 1]
```diff
@@ -177,6 +177,273 @@ _ref:
...
+
+                requests:
+                  - id: get_users
+                    type: AxiosHttp
+                    connectionId: my_api
+                    properties:
+                      url: /users
+            ```
+
+            This request should be called when the page is mounted. To do this, the `Request` action will be used during the `onMount` event of the home page.
```
Comment (Gervwyk) [inline packages/docs/concepts/connections-and-requests.yaml:224]: Change to [`Request`](/Request) .. and `onMount` event of the home page[`PageHeaderMenu`](/PageHeaderMenu) block.

File: packages/docs/concepts/connections-and-requests.yaml [hunk 2]
```diff
@@ -177,6 +177,273 @@ _ref:
...
+                      url: /users
+
+                events:
+                  onMount:
+                    - id: fetch_get_users
+                      type: Request
+                      params: get_users
+            ```
+
+            A block can be added to the page, that can use or display the data returned by this request. The `_request` operator is used to get the the results of the request.
```
Comment (Gervwyk) [inline packages/docs/concepts/connections-and-requests.yaml:254]: Again, link to _request docs `[_request](_request)`

File: packages/docs/concepts/connections-and-requests.yaml [hunk 3]
```diff
@@ -177,6 +177,273 @@ _ref:
...
+                          - _request: get_users.data
+                          - _function:
+                              label:
+                                __args: 0.name
+                              value:
+                                __args: 0.id
+            ```
+
+            To get the posts made by the selected user, a new request should be created and called once the value of the Selector changes.
+            To do this, create a new request that will make us of a `payload` to get the posts with the selected user id. Add an `onChange` event on the `Selector` block, during which the new request is called.
```
Comment (Gervwyk) [inline packages/docs/concepts/connections-and-requests.yaml:300]: *use

**Review by Gervwyk — COMMENTED** [review]
Very nice example, thank you!

File: packages/docs/concepts/connections-and-requests.yaml (hunk 1 above)
Comment (StephanieJKS) [reply packages/docs/concepts/connections-and-requests.yaml]: Change made in d611ea5901b9c9e552b83c41f6a7f5b674515f34

File: packages/docs/concepts/connections-and-requests.yaml (hunk 2 above)
Comment (StephanieJKS) [reply packages/docs/concepts/connections-and-requests.yaml]: Change made in d611ea5901b9c9e552b83c41f6a7f5b674515f34

File: packages/docs/concepts/connections-and-requests.yaml (hunk 3 above)
Comment (StephanieJKS) [reply packages/docs/concepts/connections-and-requests.yaml]: Change made in d611ea5901b9c9e552b83c41f6a7f5b674515f34

**Review by Gervwyk — APPROVED** [review]

**Pull request merged**